### ISSUE TITLE
No 'Request Entity Too Large' error when uploading too large file, upload just hangs

### ISSUE THREAD

**Issue by Kyogre** [RocketChat/Rocket.Chat#25070]

**Comment by Kyogre** [comment]
@jeanfbrito hi! If that issue affects only RC.Electron, don't you think it needs to be moved back to RC.Electron repo?

**Comment by jeanfbrito** [comment]
> @jeanfbrito hi! If that issue affects only RC.Electron, don't you think it needs to be moved back to RC.Electron repo?

This is a server issue, not a Electron issue. How it only affects Electron? This don't happens on the browser?

**Comment by Kyogre** [comment]
yes, it doesn't affects browsers. Only affects Electron. Look at the "Expected behavior" in description.

**Comment by jeanfbrito** [comment]
Yet, this is a problem in the server. The Electron is just a wrapper that opens the web client and adds some desktop functionalities.

**Comment by KevLehman** [comment]
The uploads code has been improved in newer versions, so this shouldn't happen anymore. If you encounter it again, feel free to open/reopen the issue.
